crates/csharp/src/lib.rs

```diff
@@ -1082,7 +963,7 @@ impl InterfaceGenerator<'_> {
             self.csharp_interop_src,
             r#"
             [UnmanagedCallersOnly(EntryPoint = "{export_name}")]
-            public static {wasm_result_type} {interop_name}({wasm_params}) {{
+            unsafe public static {wasm_result_type} {interop_name}({wasm_params}) {{
```

Comment:
```suggestion
            public static unsafe {wasm_result_type} {interop_name}({wasm_params}) {{
```
Also, can we drop `unsafe` ?

Reply:
> Also, can we drop unsafe ?

Not currently, a few of these still use pointers directly, we might able to clean that up by using Spans and other techniques at a later point.